azavea/tilegarden
# Remove message tiles

## Overview

`jimp` was pushing the size of the lambda function bundle over AWS Lambda's upload limit and blocking deployment attempts. Unfortunately, this makes it not worth it to have as a dependency just to support error message tiles, although they were nice. 

For reference, the breakdown of dependency size is as follows:
![image](https://user-images.githubusercontent.com/16990679/42965973-93efa2b4-8b69-11e8-9ab7-a79643f803c2.png)
The 34.1M bundle size doesn't account for mapnik (the tool I used couldn't figure out what to make of the C++ bit of it, I think), which should push the whole thing over AWS's limit of 70M (apparently).

### Notes
There are actually ways to circumvent Lambda's upload limits (such as storing the files in S3), but I didn't think adding additional infrastructural complexity was worth it for a non-essential feature.

## Testing Instructions

 * Attempt to run `./scripts/publish --new` or `./scripts/publish`, it should publish successfully (unlike on develop).

Resolves #70
